prompts/prompts-SLG.md

```
@@ -0,0 +1,70 @@
+# Prompt 1, Amazon Q
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)